test/integration/nsmd_test_utils/test_utils.go

```diff
@@ -55,18 +55,18 @@ func deployNsmdAndDataplane(k8s *kube_testing.K8s, node *v1.Node) (nsmd *v1.Pod,
 	return
 }
 
-func DeployIcmp(k8s *kube_testing.K8s, node *v1.Node) (icmp *v1.Pod) {
+func DeployIcmp(k8s *kube_testing.K8s, node *v1.Node, name string) (icmp *v1.Pod) {
```

Comment:
Can you please split this change in another patch and explain why we need it?

Reply:
Accidentally added these changes to PR, fixed